### PR TITLE
Fix invoke command

### PR DESCRIPTION
**Pull request by Tommo-L**
Close https://github.com/neo-project/neo-node/issues/614 Close https://github.com/neo-project/neo/issues/1735

Don't have to check the balance of cosiginers.

@cloud8little plz, have a check.

### CODE REVIEW

**Comment by shargon** [comment]
Yes, we only need the sender's balance

**Comment by superboyiii** [comment]
@cloud8little Could you retest your contract: https://github.com/neo-project/neo-node/issues/614

**Comment by cloud8little** [comment]
Test passed.
b189e015e8f952f969e279ee53e63ae

File: neo-cli/CLI/MainService.Contracts.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ private void OnInvokeCommand(UInt160 scriptHash, string operation, JArray contra
             {
                 using (SnapshotView snapshot = Blockchain.Singleton.GetSnapshot())
                 {
-                    UInt160[] accounts = CurrentWallet.GetAccounts().Where(p => !p.Lock && !p.WatchOnly).Select(p => p.ScriptHash).Where(p => NativeContract.GAS.BalanceOf(snapshot, p).Sign > 0).ToArray();
+                    UInt160[] accounts = CurrentWallet.GetAccounts().Where(p => !p.Lock && !p.WatchOnly).Select(p => p.ScriptHash).ToArray();
```
Comment (erikzhang) [inline neo-cli/CLI/MainService.Contracts.cs:60]: Can we use `CurrentWallet.GetAccount()` to avoid this loop?

**Comment by ProDog** [comment]
It also needs to be modified😀: 
https://github.com/neo-project/neo-node/blob/master/neo-cli/CLI/MainService.cs#L483

**Pull request merged**